content/_index.md

```diff
@@ -139,6 +139,15 @@ May Chan, CEO, <a href="https://www.hashpack.app/" target="_blank" rel="noreferr
 '''
 logo = "images/Hiero-Logo-HashPack.png"
 
+[[section_quotes]]
```

Comment:
Any idea why this is shown as addition for the PR? Can you rebase on the `main` and check if this goes away?